CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+# [1.4.0] - 2023-03-21
+
+### Changed
+
+- Common: Enable link attribute parsing
+- Common: Link button disabling logic exists for immutable links
+- Common: Backspace/delete behaviour now handles immutable links
+- iOS: Swift-tools-version bump from 5.6 to 5.7
+- Web: Selection logic now handles immutable links
+
+### Fixed
+
+- Android: Fix build for disabled link action
+
```

Comment:
```suggestion
```

Think we can remove this part as it was introduced by the `Common: Link button disabling logic exists for immutable links` change